docs/contributing.md

```diff
@@ -99,6 +101,8 @@ To build the documentation install the development environment via
 Conda.
 
 ```shell
+# replace this with environment-macos-dev.yaml or environment-windows-dev.yaml
```

Comment:
I'd prefer this to just talk about Windows. And do all mac changes in another PR.